### ISSUE TITLE
How to make artifactory:scm work with docker repository on artifactory

### ISSUE THREAD

**Issue by citicruiser** [cnenning/go-artifactory-scm-plugin#24]

**Comment by citicruiser** [comment]
hi, I just installed the latest artifactory scm plugin and I tried to make it work with artifactory docker repo.  However I have problem to make it work.   

My question is does scm plugin works with docker repository?  If yes, what is the url and "Filename Regex" I should put.  I tried several ways, but none worked for me.  I got error with below info.

Here is the setup on goCD: 
![image](https://user-images.githubusercontent.com/40603276/56616285-00674a80-65eb-11e9-9528-4c24fe2e6390.png)

and here is the error message I got when run goCD job:
![image](https://user-images.githubusercontent.com/40603276/56616307-14ab4780-65eb-11e9-805d-6a7ab1304481.png)

And here is my docker repository on artifactory:
![image](https://user-images.githubusercontent.com/40603276/56616376-3efd0500-65eb-11e9-82c5-1dfef4a29862.png)

thank you for your help.  

best,

Pei
